docs/ops.html

```diff
@@ -453,7 +459,7 @@ <h4><a id="rep-throttle" href="#rep-throttle">Limiting Bandwidth Usage during Da
   There are two interfaces that can be used to engage a throttle. The simplest, and safest, is to apply a throttle when invoking the kafka-reassign-partitions.sh, but kafka-configs.sh can also be used to view and alter the throttle values directly.
   <p></p>
   So for example, if you were to execute a rebalance, with the below command, it would move partitions at no more than 50MB/s.
-  <pre class="brush: bash;">$ bin/kafka-reassign-partitions.sh --zookeeper localhost:2181 --execute --reassignment-json-file bigger-cluster.json --throttle 50000000</pre>
+  <pre class="brush: bash;">$ bin/kafka-reassign-partitions.sh --zookeeper localhost:2181 --execute --reassignment-json-file bigger-cluster.json —throttle 50000000</pre>
```

Comment:
The help output says double-dash `--` is indeed required:

```
bin/kafka-reassign-partitions.sh
...
--throttle <Long: throttle>             The movement of partitions between
                                          brokers will be throttled to this
                                          value (bytes/sec). Rerunning with
                                          this option, whilst a rebalance is
                                          in progress, will alter the throttle
                                          value. The throttle rate should be
                                          at least 1 KB/s. (default: -1)
```

Also, direct ZooKeeper access in this tool is still supported at the moment, but it is possible that it will be deprecated in the next release as per https://issues.apache.org/jira/browse/KAFKA-8820 -- something to watch.